app/controllers/concerns/homepage_popular_links_ab_testable.rb

```diff
@@ -0,0 +1,49 @@
+module HomepagePopularLinksAbTestable
+  CUSTOM_DIMENSION = 65
```

Comment:
Please could we amend this to be 67 (https://gds.slack.com/archives/C04RFV5JC6Q/p1702542652364159?thread_ts=1702401606.332049&cid=C04RFV5JC6Q)